### PR TITLE
Update attachment_vba_macro_auto_exec_unsolicited.yml

### PR DESCRIPTION
**Pull request by morriscode**
Adding reply negation. This solves all known FP's.

### CODE REVIEW

File: response.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{"functions":["length"],"list":["recipient_domains"]}
```
Comment (jkamdjou) [inline response.txt:1]: accidental commit?

Comment (morriscode) [reply response.txt:1]: Yeah it looks like something is popping that artifact up there.

Comment (morriscode) [reply response.txt:1]: well the file's been deleted but github says it hasn't....

**Comment by morriscode** [comment]
@jkamdjou this should be gtg now.

**Comment by cameron-dunn-sublime** [comment]
@morriscode should be good now. `response.txt` is still shown here but when I checkout the branch it's not there. If it merges to main and includes the file I'll clean it up, but it won't cause problems regardless.

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**